presidio-image-anonymizer/presidio_image_anonymizer/entities/api_request_convertor.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+from typing import Tuple, Union, io
+
+from PIL import Image
+
+from presidio_image_anonymizer.entities import InvalidParamException
+
+logger = logging.getLogger("presidio-image-anonymizer")
+
+
+def get_json_data(data: str) -> dict:
+    try:
+        if not data:
+            return {}
+        return json.loads(data.replace("'", "\""))
```

Comment:
Why do we need to replace here where in the other cases we didn't?

Reply:
Since this is coming from multipart form and with -f 'data={"":""}' it is really easy to get the brackets wrong.
To save the user from frustration and tears, I prefer to insert this :)